src/blockchain.h

```diff
@@ -654,10 +654,12 @@ MOLECULE_API_DECORATOR mol_errno MolReader_Script_verify (const mol_seg_t *input
     if (input->size < MOL_NUM_T_SIZE*(field_count+1)){
         return MOL_ERR_HEADER;
     }
-    mol_num_t offsets[field_count+1];
+    if(field_count>3) return MOL_ERR_FIELD_COUNT;
+    mol_num_t offsets[4];
     offsets[0] = offset;
     for (mol_num_t i=1; i<field_count; i++) {
         ptr += MOL_NUM_T_SIZE;
+	PRINTF("-- %x\n", &i);
```

Comment:
wanted to make sure this was intentionally left in. it doesn't really hurt to have it, but could liter the logs

Reply:
It wasn't; removing it.